build.cake

```diff
@@ -146,7 +149,7 @@ class UnityPackage
 
     public void CreatePackage(string targetDirectory)
     {
-        var args = "-exportPackage ";
+        var args = "-gvh_disable -exportPackage "; // https://github.com/googlesamples/unity-jar-resolver#getting-started
```

Comment:
Please describe what `-gvh_disable` is, and why it required. Keep the link to prove it.